lib/src/main/java/com/auth0/jwt/creators/JWTCreator.java

```diff
@@ -39,9 +39,11 @@
 import java.net.URLDecoder;
 import java.net.URLEncoder;
 import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
```

Comment:
This is never used

Reply:
removed locally.  i was thinking of cleaning these up in a separate PR later.